### ISSUE TITLE
Provide a method to configure JS instrumented APIs.

### ISSUE THREAD

**Issue by englehardt** [mozilla/OpenWPM#353]
* **I'm submitting a ...**
[ ] bug report
[X ] feature request
[ ] question about the decisions made in the repository
[ ] question about how to use this project

* **Summary**

Right now it's not possible to configure the instrumented APIs without editing the content script. It would be great to be able to pass config options. One approach we've considered in the past was to pass something like: 
```
'<API Name>: {
    logging: <all, none>,
    excluded_properties: [],
    included_properties: [],
    logging_flags: [logCallBack, logFunctionsAsStrings, ...]
},
'CanvasRenderingContext2D.prototype': {
...
},
...
```

**Comment by birdsarah** [comment]
i'm working on some related stuff at the moment, so i'm self-assigning and if a pr comes out of it great. if i stop working on related stuff, i'll un-self assign.

**Comment by englehardt** [comment]
Addressed by #642
